### PR TITLE
Support user disablement, and upgrade test-containers because of a bug

### PR DESCRIPTION
**Pull request by roiravhon**

### CODE REVIEW

File: apollo-model/src/main/java/io/logz/apollo/models/User.java [hunk 1]
```diff
@@ -35,6 +36,10 @@ public boolean isAdmin() {
         return isAdmin;
     }
 
+    public boolean isEnabled() {
```
Comment (ShaharGotshtat) [inline apollo-model/src/main/java/io/logz/apollo/models/User.java]: getIsEnabled() ?

File: ui/app/scripts/config.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 
     var CONFIG = {
-        appUrl: document.location.protocol + "//" + document.location.host + "/api/",
-        // appUrl: 'http://localhost:8081/',
+        //appUrl: document.location.protocol + "//" + document.location.host + "/api/",
+        appUrl: 'http://localhost:8081/',
```
Comment (ShaharGotshtat) [inline ui/app/scripts/config.js]: Revert

File: apollo-backend/src/test/java/io/logz/apollo/AuthTest.java [hunk 3]
```diff
@@ -79,6 +79,38 @@ public void testLogin() throws Exception {
...
+        apolloTestClient.login();
+
+        // Check that its valid
+        assertThat(apolloTestClient.getAllServices()).size().isGreaterThanOrEqualTo(0);
+
+        testUser.setEnabled(false);
+        apolloTestAdminClient.updateUser(testUser);
+
+        // Check that user is forbidden even without logging in
+        assertThatThrownBy(apolloTestClient::getAllUsers).isInstanceOf(ApolloNotAuthorizedException.class);
```
Comment (ShaharGotshtat) [inline apollo-backend/src/test/java/io/logz/apollo/AuthTest.java]: Why is the request here not also getAllServices?
Why not choose a lighter request?

File: apollo-backend/src/main/java/io/logz/apollo/rest/RestServer.java [hunk 4]
```diff
@@ -85,7 +90,12 @@ private void registerLoginProvider() {
     private void registerRolesProvider() {
         My.rolesProvider((req, username) -> {
             try {
-                if (userDao.getUser(username).isAdmin()) {
+                User user = userDao.getUser(username);
+                if (!user.isEnabled()) {
+                    req.response().code(HttpStatus.FORBIDDEN);
+                    return null;
```
Comment (ShaharGotshtat) [inline apollo-backend/src/main/java/io/logz/apollo/rest/RestServer.java]: you can "return;" without the null

File: ui/app/scripts/config.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 
     var CONFIG = {
-        appUrl: document.location.protocol + "//" + document.location.host + "/api/",
-        // appUrl: 'http://localhost:8081/',
+        //appUrl: document.location.protocol + "//" + document.location.host + "/api/",
```
Comment (asafalima) [inline ui/app/scripts/config.js]: revert

File: apollo-model/src/main/java/io/logz/apollo/models/User.java (hunk 1 above)
Comment (roiravhon) [reply apollo-model/src/main/java/io/logz/apollo/models/User.java]: No, that's the boolean convention

File: apollo-backend/src/test/java/io/logz/apollo/AuthTest.java (hunk 3 above)
Comment (roiravhon) [reply apollo-backend/src/test/java/io/logz/apollo/AuthTest.java]: good catch

File: apollo-backend/src/main/java/io/logz/apollo/rest/RestServer.java (hunk 4 above)
Comment (roiravhon) [reply apollo-backend/src/main/java/io/logz/apollo/rest/RestServer.java]: I can't. RolesProvider expect a Set

**Pull request merged**